tests/include/aws_test_tcp.h

```diff
@@ -30,10 +30,10 @@
  * Update udptestECHO_SERVER_ADDR# and
  * udptestECHO_PORT with IP address
  * and port of unencrypted TCP echo server. */
-#define udptestECHO_SERVER_ADDR0         10
-#define udptestECHO_SERVER_ADDR1         184
-#define udptestECHO_SERVER_ADDR2         166
-#define udptestECHO_SERVER_ADDR3         154
+#define udptestECHO_SERVER_ADDR0         34
```

Comment:
Why is an IP address hard-coded in here?

Reply:
We do have hard address for TCP echo server  as well. (see below tcptestECHO_SERVER_ADDR0 ...) this file is for the test infra and hence has a hard coded address for testing with our test server (although not the best design), Ideally this server should be the same  as TCP echo server, but running UDP tests internally with external EC2 machine does not work.

Reply:
This is common library code and its tests should have no dependencies on any particular HW lab environment or setup.

Reply:
This IP is public.

Reply:
Squashed commit so internal test UDP echo server address is not in record